Github app shows as not installed after installing

I'm trying to migrate a project as instructed. When I get to this page I click on Install

<img width="784" height="411" alt="Image" src="https://github.com/user-attachments/assets/8a47ecc5-22f7-4864-a169-26b2ac1673e2" />

but Github already says that "Read the Docs Community is already installed as suggested"

<img width="1400" height="923" alt="Image" src="https://github.com/user-attachments/assets/d1667eb9-dea9-4aa7-8c12-3cc2549b45e6" />

I've clicked on Refresh a dozen times, nothing changes.

Am I doing something wrong?

I removed the Github app, still showing in RDT and can not remove it. Can you remove it from your side?


This is the console error when trying to dismiss the notification

```
vendor.fe09765eb8a5.js:281  PATCH https://app.readthedocs.org/ 405 (Method Not Allowed)
(anonymous) @ vendor.fe09765eb8a5.js:281
value @ site.7fcfa6207486.js:3
handleEvent @ vendor.fe09765eb8a5.js:310
i @ vendor.fe09765eb8a5.js:281
vendor.fe09765eb8a5.js:281 Error dismissing notification Error: Invalid API request
    at site.7fcfa6207486.js:3:25378
(anonymous) @ vendor.fe09765eb8a5.js:281
(anonymous) @ site.7fcfa6207486.js:3
Promise.catch
value @ site.7fcfa6207486.js:3
handleEvent @ vendor.fe09765eb8a5.js:310
i @ vendor.fe09765eb8a5.js:281
vendor.fe09765eb8a5.js:281 Fetch failed loading: PATCH "https://app.readthedocs.org/".
(anonymous) @ vendor.fe09765eb8a5.js:281
value @ site.7fcfa6207486.js:3
handleEvent @ vendor.fe09765eb8a5.js:310
i @ vendor.fe09765eb8a5.js:281
```

--- a/readthedocs/api/v3/serializers.py
+++ b/readthedocs/api/v3/serializers.py
@@ -263,7 +263,7 @@ class NotificationSerializer(serializers.ModelSerializer):
         read_only_fields = ["dismissable", "news"]
 
     def get_attached_to_content_type(self, obj):
-        return obj.attached_to_content_type.name
+        return obj.attached_to_content_type.model
 
 
 class VersionLinksSerializer(BaseLinksSerializer):

--- a/readthedocs/api/v3/serializers.py
+++ b/readthedocs/api/v3/serializers.py
@@ -263,6 +263,7 @@ class NotificationSerializer(serializers.ModelSerializer):
         read_only_fields = ["dismissable", "news"]
 
     def get_attached_to_content_type(self, obj):
+        # NOTE: Don't user name, because it can change based on the current language.
         return obj.attached_to_content_type.model
 
 

--- a/readthedocs/notifications/models.py
+++ b/readthedocs/notifications/models.py
@@ -87,11 +87,10 @@ class Notification(TimeStampedModel):
         return message
 
     def get_absolute_url(self):
-        content_type_name = self.attached_to_content_type.name
-        path = ""
+        content_type_name = self.attached_to_content_type.model
         if content_type_name == "user":
             url = "users-notifications-detail"
-            path = reverse(
+            return reverse(
                 url,
                 kwargs={
                     "notification_pk": self.pk,
@@ -99,10 +98,10 @@ class Notification(TimeStampedModel):
                 },
             )
 
-        elif content_type_name == "build":
+        if content_type_name == "build":
             url = "projects-builds-notifications-detail"
             project_slug = self.attached_to.project.slug
-            path = reverse(
+            return reverse(
                 url,
                 kwargs={
                     "notification_pk": self.pk,
@@ -111,10 +110,10 @@ class Notification(TimeStampedModel):
                 },
             )
 
-        elif content_type_name == "project":
+        if content_type_name == "project":
             url = "projects-notifications-detail"
             project_slug = self.attached_to.slug
-            path = reverse(
+            return reverse(
                 url,
                 kwargs={
                     "notification_pk": self.pk,
@@ -122,13 +121,14 @@ class Notification(TimeStampedModel):
                 },
             )
 
-        elif content_type_name == "organization":
+        if content_type_name == "organization":
             url = "organizations-notifications-detail"
-            path = reverse(
+            return reverse(
                 url,
                 kwargs={
                     "notification_pk": self.pk,
                     "parent_lookup_organization__slug": self.attached_to.slug,
                 },
             )
-        return path
+
+        raise ValueError("Unknown content type for notification")

--- a/readthedocs/notifications/models.py
+++ b/readthedocs/notifications/models.py
@@ -87,6 +87,7 @@ class Notification(TimeStampedModel):
         return message
 
     def get_absolute_url(self):
+        # NOTE: Don't user name, because it can change based on the current language.
         content_type_name = self.attached_to_content_type.model
         if content_type_name == "user":
             url = "users-notifications-detail"

